etc/kayobe/networks.yml

```diff
@@ -59,6 +59,12 @@
 # hosts
 #cleaning_net_name:
 
+# Name of the network used to manage octavia loadbalancers
+octavia_net_name: ""
+
+# Name of the network used to manage network switches
+switch_mgmt_net_name: ""
```

Comment:
Can remove this and rules referring to it

Reply:
Done